core/app/models/spree/google_export_option.rb

```diff
@@ -0,0 +1,19 @@
+module Spree
+  class GoogleExportOption < Base
+    def export
```

Comment:
I would explicitly name it `export_rss_feed`, if you just name it export it would imply that you are exporting the options themselves.
```suggestion
    def export_rss_feed
```
I'm not 100% sure about placing this method in this class -- should the feed settings class also be responsible for generating the feed itself? That being said, we don't have a class for the feed itself, and it definitely does not need to be a resource if it doesn't have a corresponding db table.

Another option is to have the store model/controller handle this functionality (for example, as a module that gets included in the store model or controller), unless we plan on allowing each store to have multiple collections of settings for different cases.

@przemosk @wjwitek thoughts?

Reply:
> I'm not 100% sure about placing this method in this class -- should the feed settings class also > be responsible for generating the feed itself? That being said, we don't have a class for the feed > itself, and it definitely does not need to be a resource if it doesn't have a corresponding db table.

@nciemniak fair point! I missed that 😇 we should move this behaviour from here 

What I would suggest is: 
1. remove method export from here
2. move service object into controller and execute there
3. in controller, variable `@options` can be set before action `show` by adding `callback` ( optional )

fyi: @wjwitek

Reply:
If I understand correctly I should just move the whole export function to the controller in api?

Reply:
@nciemniak @przemosk forgot to tag you :)

Reply:
yep, correctly 👌🏻

Reply:
Or to move to the api's store controller? I suppose it would make sense to have it there.